src/styles/global.css

```
@@ -229,4 +271,553 @@ h3 {
+
+.chat-new-chat {
+  display: block;
+  text-decoration: none;
+  padding: 0.65rem 0.75rem;
+  border-radius: 0.6rem;
+  border: 1px solid var(--chat-border);
+  color: var(--text-primary);
+  font-weight: 600;
+  text-align: center;
+  background: rgba(255, 255, 255, 0.02);
+  display: flex;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)